Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -1347,6 +1345,28 @@ ErrorOr<void> MainWidget::create_image_from_clipboard()
     return {};
 }
 
+void MainWidget::refresh_crop_to_selection_menu_actions()
+{
+    size_t i = 0;
+    GUI::Action* menu_action = nullptr;
+    for (i = 0; i < m_image_menu->items().size(); i++) {
+        menu_action = m_image_menu->action_at(i);
+        if (menu_action && menu_action->text().contains("Crop "sv) && menu_action->text().contains(" to Selection"sv)) {
```

Comment:
Instead of walking the menu and matching on the action text, what do you think about storing the result of `GUI::Action::create` for the two actions in a member variable and then calling the method in the action directly?

Reply:
done